src/librustc_data_structures/frozen.rs

```diff
@@ -0,0 +1,57 @@
+//! An immutable, owned value.
```

Comment:
Up to interior mutability (unless you want to use `Freeze` to exclude those).

Reply:
Yes, I updated the docs.